datajoint/user_relations.py

```diff
@@ -2,59 +2,83 @@
 Hosts the table tiers, user relations should be derived from.
 """
 
-import abc
 from .base_relation import BaseRelation
 from .autopopulate import AutoPopulate
 from .utils import from_camel_case
 from . import DataJointError
 
 
-class Part(BaseRelation, metaclass=abc.ABCMeta):
+class classproperty:
```

Comment:
Why are we reintroducing `classproperty`? You have ditched this idea in the past since the concept of `classproperty` is not a native one to Python.